_data/projects/wordview.yml

```diff
@@ -0,0 +1,12 @@
+name: Wordview
+desc: A Python package for Exploratory Data Analysis (EDA) for text-based data.
+site: https://github.com/meghdadFar/wordview
+tags:
+  - nlp
+  - text-analysis
+  - exploratory-data-analysis
+  - multiword-expressions
+  - anomalies
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/meghdadFar/wordview/labels/up%20for%20grabs
```

Comment:
There's no label on your project matching `up for grabs` - please update this URL and the `name` value to a list of curated issues which are ideal for new contributors...

Reply:
@shiftkey Thanks for your review. 
Fixed.